meetings/rrsagent.md

```diff
@@ -0,0 +1,206 @@
+
+## Talking to RRSAgent {#talking}
+
+RRSAgent also recognizes several commands during its logging session. Most commands begin with '`rrsagent, `', though a few are recognized without this.
+
+### Action Item Tracking {#actions}
+
+RRSAgent will help track action items recorded while a meeting is in progress. Many of the commands in this set need not be directly addressed to RRSAgent; they will be recognized when addressed to no one in particular and also when received as third person ('`/me`') messages.
+
+The list of action items may be reviewed and modified during the meeting. The action items will be recorded as they are modified in `https://www.w3.org/yyyy/mm/dd-<channel>-actions` where where '`yyyy`' is the current year, '`mm`' is the current month, and '`dd`' is the current day.
+
+- `[rrsagent,] ACTION: <text>`
+
+Adds an action item described by `<text>`, assigning it the next sequential number. This command need not be addressed to RRSAgent directly. The keyword '`ACTION`' is not case sensitive. Actions may not be assigned in '`/me`' messages.
+
+- `[rrsagent,] ACTION <number> = <newtext>`
+
+Replaces the text of action item `<number>` with `<newtext>`. This need not be addressed to RRSAgent directly but will be ignored in '`/me`' lines. The keyword '`ACTION`' must be in all-caps as shown or all lowercase.
+
+- `[rrsagent,] ACTION- <number>`
+
+Removes action item `<number>` from the list. This need not be addressed to RRSAgent directly. The keyword '`ACTION`' is not case sensitive..
+
+- `rrsagent, what [are the] action items?`
+- `rrsagent, what [are the] actions?`
+- `rrsagent, [please] show [the] action items`
+- `rrsagent, [please] show [the] actions`
+- `rrsagent, [please] list [the] action items`
+- `actions?`
+
+Displays the list of action items. The first set of requests must be addressed specifically to RRSAgent. The last request will respond in '`/me`' (third-person or out-of-band) form.
+
+- `rrsagent, [please] drop action <number>`
+
+Removes action item `<number>` from the list. This form must be addressed specifically to RRSAgent
+
+- `rrsagent, [please] ignore action items`
+- `rrsagent, [please] ignore actions`
+
+Requests that RRSAgent stop tracking action items. This request must be addressed specifically to RRSAgent
+
+- `rrsagent, [please] track action items`
+- `rrsagent, [please] track actions`
+
+Requests that RRSAgent begin tracking action items. This request must be addressed specifically to RRSAgent. This is the default state when RRSAgent is invited to a channel
+
+### Searching the Log {#search}
+
+- `rrsagent, bookmark`
+- `rrsagent, pointer`
+- `rrsagent, here`
+- `rrsagent, where am I?`
+
+Reports a URI to the current location in the log
+
+- `rrsagent, grep [-i] [first-last|max] <text>`
+- `rrsagent, search for [-i] [first-last|max] <text>`
+- `rrsagent, find [-i] [first-last|max] <text>`
```

Comment:
@swickr I believe that `first-last` is a typo and should be `first|last`. DO you confirm?

```suggestion
- `rrsagent, grep [-i] [first|last|max] <text>`
- `rrsagent, search for [-i] [first|last|max] <text>`
- `rrsagent, find [-i] [first|last|max] <text>`
```